ethindex/logdecode.py

```diff
@@ -41,8 +41,10 @@ def add_abi(contract_address, contract_name):
     for network in addresses_json["networks"]:
         add_abi(network, "CurrencyNetwork")
 
-    add_abi(addresses_json["unwEth"], "UnwEth")
-    add_abi(addresses_json["exchange"], "Exchange")
+    if "unwEth" in addresses_json:
```

Comment:
if you are on it maybe also check if networks are there. COuld be in theory that some one wants to list no network, even tho that doesnt make too much sense.

Reply:
he can just pass an empty list in that case.